cmake/modules/CheckDependentLibraries.cmake

```diff
@@ -256,28 +256,39 @@ if(WIN32)
   set(BLA_PREFER_PKGCONFIG ON)
   set(BLA_PKGCONFIG_BLAS "openblas")
   set(BLA_PKGCONFIG_LAPACK "openblas")
-endif()
-
-if(WITH_BLAS)
-  find_package(BLAS REQUIRED)
-  if(BLAS_FOUND)
-    add_library(BLAS INTERFACE IMPORTED GLOBAL)
-    set_property(TARGET BLAS PROPERTY INTERFACE_LINK_LIBRARIES
-                                      ${BLAS_LIBRARIES})
-    set_property(TARGET BLAS PROPERTY INTERFACE_INCLUDE_DIRECTORIES
-                                      ${BLAS_INCLUDE_DIRS})
+else()
+  set(BLA_PKGCONFIG_BLAS "blas-netlib")
+  set(BLA_PKGCONFIG_LAPACK "lapacke")
```

Comment:
@nilason How can we better support OpenBLAS? I cannot compile with BLAS and LAPACK using OpenBLAS on Linux. `FindBLAS` can find different libraries, but this change doesn't use it. Was this change to avoid a custom FindBLAS.cmake?

https://cmake.org/cmake/help/latest/module/FindBLAS.html